_includes/tutorials/dynamic-output-topic/kstreams/markup/dev/make-topology.adoc

```diff
@@ -0,0 +1,41 @@
+////
+In this file you describe the Kafka streams topology, and should cover the main points of the tutorial.
+The text assumes a method buildTopology exists and constructs the Kafka Streams application.  Feel free to modify the text below to suit your needs.
+////
+
+
+The focus of this tutorial is using attributes in the output records to determine or choose the correct output topic to send to.  For sending fully processed records typically you would use the `KStream.to()` method which takes the name of the output topic where to processed records are sent. But you set this topic name when building the topology, which is fine in many cases, but what if you want to route your processed events to different output topics at runtime?
+
+For dynamic output topic choice, Kafka Streams has an overloaded version of the `KStream.to()` method that takes a `TopicNameExtractor` interface instead of a singular topic name.  The `TopicNameExtractor` interface contains only one method `extract`, which means you can use a lambda in most cases instead of a concrete class.
```

Comment:
```suggestion
For dynamic output topic choice, Kafka Streams has an overloaded version of the `KStream.to()` method that takes a `TopicNameExtractor` interface instead of a singular topic name.  The `TopicNameExtractor` interface contains only one method, `extract`. This means you can use a lambda in most cases, instead of a concrete class.
```